### PR TITLE
Fix SSL connections issue

### PR DESCRIPTION
**Pull request by GillesRasigade**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,8 @@
     "sinon": "^1.17.3"
   },
   "dependencies": {
-    "amqplib": "^0.4.0",
-    "chpr-logger": "2.4.1",
+    "amqplib": "^0.5.0",
+    "chpr-logger": "^2.4.1",
```
Comment (alexjab) [inline package.json]: Remove `^`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chauffeur-prive/node-amqp-bus",
-  "version": "4.1.0",
+  "version": "4.2.0",
```
Comment (alexjab) [inline package.json]: Bump major (out of safety)

File: lib/client.js [hunk 3]
```diff
@@ -12,14 +13,18 @@ const DEFAULT_HEARTBEAT = 10;
...
-function* createClient(url, options) {
+function* createClient(rabbitmqUrl, options) {
   options = options || {};
   options.heartbeat = options.heartbeat || DEFAULT_HEARTBEAT;
-  const connection = yield amqplib.connect(url, options);
+
+  const parsedurl = url.parse(rabbitmqUrl);
+  options.servername = parsedurl.hostname;
+
+  const connection = yield amqplib.connect(rabbitmqUrl, options);
```
Comment (alexjab) [inline lib/client.js]: Maybe add a comment with the url where you saw this trick

File: package.json (hunk 2 above)
Comment (GillesRasigade) [inline package.json]: This is not a major release from my opinion, more a bug fixing and possibly connectivity issue

Comment (alexjab) [inline package.json]: We'll do it with Mohammed at some point.

**Pull request merged**